### ISSUE TITLE
Add to Terminal.md

### ISSUE THREAD

**Issue by DishantIsrani** [HarshKapadia2/cli#8]
You can add the ```wget``` command as well.
I hope this helps - https://linuxize.com/post/wget-command-examples/

**Comment by HarshKapadia2** [comment]
Thank you for raising the issue @DishantIsrani!
Would you like to open a pull request to add that?

**Comment by DishantIsrani** [comment]
> Thank you for raising the issue @DishantIsrani! Would you like to open a pull request to add that?

Yes definitely! I will start working on it!

**Comment by HarshKapadia2** [comment]
Great! Looking forward to seeing a PR from you!

**Comment by DishantIsrani** [comment]
I have done a PR! Please check it and let me know if there are any changes required!

**Comment by HarshKapadia2** [comment]
Hey @DishantIsrani, I checked out the PR.
You have literally copy-pasted the contents from link that you shared. Can you please modify things as per your understanding and use easier examples?

**Comment by DishantIsrani** [comment]
Yes, I have actually used the site as well as my own knowledge. I will still try to make some changes according to what you have suggested!

**Comment by DishantIsrani** [comment]
I have made some changes and updated it. I hope you can check it and update me if there is still some changes required!

**Comment by HarshKapadia2** [comment]
Any suggestions @KartikSoneji?
